<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vTableWidget</class>
 <widget class="QWidget" name="vTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTableWidget" name="tbl"/>
   </item>
   <item>
    <widget class="QWidget" name="wdgPagination" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Row Per Page:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbRowPerPage">
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFirst">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPrevious">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spnPageNumber">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string> / 0</string>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNext">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLast">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
